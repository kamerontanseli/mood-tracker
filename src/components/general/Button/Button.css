.Button {
  padding: 10px 20px;
  border-radius: 4px;
  border: 0;
  background-color: #eee;
  color: #333;
}

.Button:disabled {
  opacity: 0.7;
}

.Button:focus {
  outline: none;
}

.Button:hover {
  cursor: pointer;
  background-color: #ddd;
}