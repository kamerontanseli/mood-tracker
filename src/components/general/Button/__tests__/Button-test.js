import React from 'react'
import renderer from 'react-test-renderer'
import Button from '..'

describe('render', () => {
  it('should render', () => {
    const snapshot = renderer.create(
      <Button>Hello world</Button>
    );
    expect(snapshot).toMatchSnapshot()
  })
})