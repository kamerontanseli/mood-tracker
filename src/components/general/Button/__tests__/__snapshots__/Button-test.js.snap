// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render should render 1`] = `
<button
  className="Button"
>
  Hello world
</button>
`;
